From 9ace2546b6e1b11c33019bb4a157eb91cb267fa5 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Wed, 16 Sep 2020 22:38:11 +0200
Subject: [PATCH 1/2] Don't use USE_BCRYPTGENRANDOM for VS older than 2015

---
 crypto/rand/rand_win.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/crypto/rand/rand_win.c b/crypto/rand/rand_win.c
index 9036546008..a65ecf461a 100644
--- a/crypto/rand/rand_win.c
+++ b/crypto/rand/rand_win.c
@@ -19,7 +19,7 @@
 
 # include <windows.h>
 /* On Windows Vista or higher use BCrypt instead of the legacy CryptoAPI */
-# if defined(_MSC_VER) && _MSC_VER > 1500 /* 1500 = Visual Studio 2008 */ \
+# if defined(_MSC_VER) && _MSC_VER >= 1900 /* 1900 = Visual Studio 2015 */ \
      && defined(_WIN32_WINNT) && _WIN32_WINNT >= 0x0600
 #  define USE_BCRYPTGENRANDOM
 # endif
-- 
2.28.0

